### PR TITLE
Bug 1915907: replace test/cmd/builds.sh direct/indirect docker.io refs with imagestream image refs

### PR DESCRIPTION
**Pull request by gabemontero**
note at least some of these files were touched by clayton's huge refactor, but a few refs slipped the cracks

/assign @bparees 
(in Adam's absence)

thanks @dmage for the heads up

### CODE REVIEW

**Comment by bparees** [comment]
moves in the right direction, so
/lgtm

but ultimately don't we want these refs to all be substituted so the tests can work in a disconnected env too?

**Comment by gabemontero** [comment]
> moves in the right direction, so
> /lgtm
> 
> but ultimately don't we want these refs to all be substituted so the tests can work in a disconnected env too?

Yes we do, and it occurred to me, but I was initially worried if permissions to pull from the internal image registry directly were available to these cmd tests.

That said, I've send found some of those refs so presumably they worked before.

So I'll 

/hold

for this for now and inspect the test results.  Assuming those internal image registry refs are in fact successfully used, I'll go back and change this along those lines.

thanks @bparees

File: test/extended/testdata/bindata.go [hunk 1]
```diff
@@ -30833,8 +30833,8 @@ os::test::junit::declare_suite_start "cmd/builds"
 template='{{with .spec.output.to}}{{.kind}} {{.name}}{{end}}'
 
 # Build from Dockerfile with output to ImageStreamTag
-os::cmd::expect_success "oc new-build --dockerfile=\$'FROM centos:7\nRUN yum install -y httpd'"
-os::cmd::expect_success_and_text "oc get bc/centos --template '${template}'" '^ImageStreamTag centos:latest$'
+os::cmd::expect_success "oc new-build --to=tests:custom--dockerfile=\$'FROM image-registry.openshift-image-registry.svc:5000/openshift/tests:latest\nRUN yum install -y httpd'"
```
Comment (xiuwang) [inline test/extended/testdata/bindata.go:30836]: no space between two options `--to=tests:custom--dockerfile`

**Comment by gabemontero** [comment]
OK agnostic CMD is now passing !  Think I've replace docker.io refs with imagestream docker refs

also should run

/test e2e-aws-builds
/test e2e-gcp-builds

in case those changed json's are used in the builds e2e's

**Comment by gabemontero** [comment]
hit the situation cited in slack announce-testplatform with first e2e-gcp-builds run ... retrying per directions there

/test e2e-gcp-builds

**Comment by gabemontero** [comment]
ok builds passes too ... squashing commit and releasing hold

**Comment by gabemontero** [comment]
/hold cancel

ready for re-review @bparees

**Pull request merged**